src/ut_kvp.c

```diff
@@ -165,7 +167,9 @@ static ut_kvp_status_t ut_kvp_getField(ut_kvp_instance_t *pInstance, const char
         return UT_KVP_STATUS_NO_DATA;
     }
 
-    snprintf( zEntry, UT_KVP_MAX_ELEMENT_SIZE, "%s %s", pszKey, str );
+    convert_dot_to_slash(pszKey, zKey);
+
+    snprintf( zEntry, (UT_KVP_MAX_ELEMENT_SIZE * 2), "%s %s", zKey, str );
```

Comment:
`snprintf( zEntry, (UT_KVP_MAX_ELEMENT_SIZE * 2), "%s %s", zKey, str );`

should be

`snprintf( zEntry, sizeof(zEntry), "%s %s", zKey, str );`